ray-curator/ray_curator/models/pos_embed.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) OpenGVLab. All rights reserved.
```

Comment:
@sarahyurick are models top level or inside models/video and so on? what if a model is multi-modal?

Reply:
Yeah we need to converge. Are there any multi-modal models that we are supporting for this release (I think for video and image)?